ethansiegl/matchminer-ui
# OS updates (#8)

* merged EAP updates

* EAP backend changes

* EAP revisions round 2

* EPIC UI changes requested after meeting 6/14

* hide elements on clinicaltrials overview page for restricted EPIC view

* EPIC button flagging

* EPIC error message

* EPIC error message for unauthorized users

* refactor - move epic flag to cookie from URL param

* refactor - move epic flag to cookie from URL param

* refactor - move epic flag to cookie from URL param - iteration 2

* change comment

* revert cookie EPIC code

* epic error page refactor to work correctly

* test redirect $digest bug fix in staging

* revert previous commit, move deep linking code to request.interceptor

* deep linking fix to not error

* OKTA redirect issue in EPIC causing IE10 issues. Bypass SAML authentication in that case and only verify with MM's db

* revert OKTA code

* pull out afterAuthRedirect for epic testing

* EPIC debugging, no PIWIK, 404 disable

* revert last changes

* merge possible epic final fixes

* update docker file, remove extraneous

* update dockerfile to work with new devops

* .

* update docker ignore file

* - remove copy command. /dist to be mounted through docker-compose file

* docker comments

* reverted

* - remove dockerfile, move to devops repo
- update staging elasticsearch index to use 'staging'
- rename staging to 'stage' and production to 'prod' to work more seamlessly with devops script
- move dockerfile into mmdevops repo
- change staging url

* update piwik url

* update piwik url

* force cache update

* remove overriding of default elasticsearch index val

* update default elasticsearch index

* update default elasticsearch index (2)

* update piwik tracking url

* remove early adopter pages and related js

* update help email for epic errors

* update help email for epic errors

* add js source map

* remove vendor.js from source map build step

* comment out auto scroll

* add source map

* add null checker for drug tags
uncomment out other scroll checker

* single equals

* fix

* clinical trial page loading all trials bugfix - maybe

* clinical trial page loading all trials bugfix - take 2

* add logic to only clear patient search size when navigating to clinicaltrials from patient page

* revert last commit

* update readme

* what the hell gitlabs...
* remerge missing commits

* add jenkinsfile

* update jenkinsfile

* update jenkinsfile

* update jenkinsfile

* update jenkinsfile

* update jenkinsfile

* update jenkinsfile

* update jenkinsfile - grunt

* update jenkinsfile - grunt

* update jenkinsfile - grunt

* get unit tests to run + jenkinsfile + xml report

* add banner

* update banner message

* update banner message

* remove banner

* - update dev config to point to staging elasticsearch index
- update dev config to use local http (not https) dev server
- update readme

* Update README

* update elastic dev index

* - default dev env to use remote api and add readme

* - default dev env to use remote api and add readme (2)

* - update readme

* .

* .

* .

* add null check for dev data

* update results viewer link to point to generic page istead of broken link

* updated README

* add package-log.json to version control

* - update SV text in filter popup
- add table to actionable oncopanel page
- add table to additional oncopanel page
- add coordinator email to clinical trials details page

* update readme

* add PDF layout version 3 + html templates and tooltips

* - add tooltip
- remove console.log statement

* SV + signatures formatting

* trial accrual status => trial curation level status

* add chips for signatures

chips on trial details summary

trial tree chips

chips n stuff

move signatures to mut sig category

trial accrual update

variant table update

* update badge color

* get all signatures bug fix for trial summary page

* fix unit tests

* add TMB badges to trial tree

* bugfix for badges

* bugfix for badges

* make advanced search work for temolzolmide drug and signature

* advanced search elasticsearch bugfix

* temozolomide UI test

* oncopanel table fixes

* update tooltip

* add red color to abandoned trials

* add tooltip to oncopanel page

* add tooltip to trial match for SV's

* add show_in_ui: true to query for trial matches

* simplify

* fix unit tests

* make structured SV only appear for PDF layout version 3

* layout 2/3 fix

* formatting fix for signature

* formatting fix for trial details page status badge + icon

* typo

* update tooltip on trial match page

* capitalize TMV in genomic match column on trial matches page

* do not display gene/variant on trial matches page if trial match is an SV

* display any_gene for trial curations in trial match tree and in genomic alterations table

* - hide tier values from TMB and other signature matches
- remove extra title case code
- add TMB tooltip

* - all oncopanel v2 should also show for v3
- move structured SV to new location

* - update oncopanel 'dna variants' text to 'mutations'

* mutational burden button in oncopanel sidebar - relevant for pdf_layout_version 2 and 3

* update tooltips on the trial match page

* trial match tooltip updates

* update structured SV table

* tool tip fixes - trial match page

* text update for when no structured SV data is available

* update structured SV text

* update structural variant tooltips

* fix null null issue in oncopanel structured sv tables

* keep variant tooltips from showing up for SV data

* fix tooltips for oncopanel mutations and structured svs

* typo

* update tooltips

* update tooltips

* fix bug where when matches don't have a signature code breaks

* structural variations -> structural variants

* don't return trial matches where is_disabled is flagged as true

* move trial status filtering to the UI from API

* fix unit tests

* temp rollback of filter related UI updates

* ie11 ie8 document mode bugfix

* revert previous change

* faq update

* temporarily disable patient matches and filter pages for filter fixing

* reenable filters and patient matches in navbar

* re-enable SV filter matching updates

* add back dockerfile

* update readme, properties file

* update properties

* updates
